### PR TITLE
Dotenv Variables 

### PR DESCRIPTION
**Pull request by LamaDabbeet**
The third task is done.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,32 +0,0 @@
-const express = require('express')
```
Comment (NourSammour) [inline server.js]: we should use both dot-env and express server, why did you remove it

**Review by NourSammour — CHANGES_REQUESTED** [review]

File: server.js (hunk 1 above)
Comment (LamaDabbeet) [reply server.js]: I have merged both dot-env and express server.

File: pages/about.js [hunk 2]
```diff
@@ -1 +0,0 @@
-export default () => <div>About us</div>
```
Comment (NourSammour) [inline pages/about.js]: why did you remove this files?

Comment (LamaDabbeet) [reply pages/about.js]: I have added it

File: pages/day/index.js [hunk 3]
```diff
@@ -1 +0,0 @@
-export default () => <div>Hello Day</div>
```
Comment (NourSammour) [inline pages/day/index.js]: why did you remove this?

**Review by NourSammour — CHANGES_REQUESTED** [review]

**Review by NourSammour — APPROVED** [review]

**Pull request merged**